src/cli/configure/analyseConfiguration.ts

```diff
@@ -0,0 +1,48 @@
+import path from 'path';
+
+import fs from 'fs-extra';
+
+import { log } from '../../utils/logging';
+
+import { diffFiles } from './analysis/project';
+
+interface Props {
+  destinationRoot: string;
+  entryPoint: string;
+}
+
+export const analyseConfiguration = async (
```

Comment:
This is just a cut down version of `src/cli/configure/applyConfiguration.ts` as some logic was lifted to the `index.ts` function